File: AplicacionPeliculas/TablasIntermedias.vb
Public Class TablasIntermedias
    Dim idbuscaP As Integer
    Private Sub PicBuscaPeli_Click(sender As Object, e As EventArgs) Handles PicBuscaPeli.Click
        Peliculas.Show()
    End Sub

    Private Sub TablasIntermedias_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        idbuscaP = Peliculas.idpeli
    End Sub


End Class

File: AplicacionPeliculas/TablasIntermedias.cs
using System;

namespace AplicacionPeliculas
{
    public partial class TablasIntermedias
    {
        private int idbuscaP;

        public TablasIntermedias()
        {
            InitializeComponent();
        }
        private void PicBuscaPeli_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Peliculas.Show();
        }

        private void TablasIntermedias_Load(object sender, EventArgs e)
        {
            idbuscaP = My.MyProject.Forms.Peliculas.idpeli;
        }


    }
}